images/hub/requirements.txt

```diff
@@ -1,3 +1,4 @@
+jhub_remote_user_authenticator==0.1.0
```

Comment:
This PR adds an additional authenticator that can't be used with this chart. Can you explain why this is added here without any support for using it?